CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### :magic_wand: Added
 - Host Availability Strategy to help keep host health status up to date ([PR #530](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/530)).
 
+### :bug: Fixed
+- Race conditions between `MonitorThreadContainer#getInstance()` and `MonitorThreadContainer#releaseInstance()` ([PR #601](https://github.com/awslabs/aws-advanced-jdbc-wrapper/pull/601)).
```

Comment:
NIT: Fixed a case of race condition between...

Reply:
None of the Fixed changelog items start with "Fixed ...." and instead either state the bug, or what was done to fix the bug. I did this once and was called out and IIRC the reason was that "Fixed..."  is redundant with the header.

Reply:
Done!